.github/workflows/linting.yml

```diff
@@ -2,13 +2,12 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: Actions
+name: linting
 
 on:
   push:
-    branches: [master, main]
+    branches: [main]
   pull_request:
-    branches: [master, main]
```

Comment:
Dus nu ook op pull request van branch A naar branch B, waarbij B niet de main is?

Reply:
Ja, hij lint op alle PRs